index.php

```diff
@@ -97,14 +97,16 @@
             var layer_tidalscale;                  // 16
             var layer_permalink;                   // 17
             var layer_waterdepth_trackpoints_100m;      // 18
+			 // var layer_elevation_profile_contours;  // 19
```

Comment:
Can you harmonize the indentation in this line?

Reply:
I'll merge once the indentation is in-line.

Reply:
@wschildbach Just fixed it.

Reply:
Thanks @OpenDEM , this is now fixed. @Nautic , FYI.